packages/client/modules/meeting/components/AddPokerTemplateDimension.tsx

```diff
@@ -63,16 +63,14 @@ class AddPokerTemplateDimension extends Component<Props> {
     )
   }
 
-  render() {
-    const {dimensions, submitting} = this.props
-    if (dimensions.length >= Threshold.MAX_REFLECTION_PROMPTS) return null
-    return (
-      <AddDimensionLink palette='blue' onClick={this.addPrompt} waiting={submitting}>
-        <AddDimensionLinkPlus>add</AddDimensionLinkPlus>
-        <div>Add another dimension</div>
-      </AddDimensionLink>
-    )
-  }
+  const {dimensions, submitting} = props
+  if (dimensions.length >= Threshold.MAX_REFLECTION_PROMPTS) return null
+  return (
+    <AddDimensionLink palette='blue' onClick={addDimension} waiting={submitting}>
+      <AddDimensionLinkPlus>add</AddDimensionLinkPlus>
+      <div>Add another dimension</div>
+    </AddDimensionLink>
+  )
 }
 
 export default createFragmentContainer(withMutationProps(withAtmosphere(AddPokerTemplateDimension)), {
```

Comment:
ah, for the refactor can you repace the HOCs with hooks? 
e.g. get rid of `withAtmosphere` and add `useAtmosphere`